src/components/Quotes/Quotes.jsx

```diff
@@ -7,7 +7,7 @@ import styleDark from '../Quotes/Light.css';
 
 
 const Quotes = ({ quotes }) => {
-  const quotesAttributes = quotes.map(quote =>(
+  const quotes = quotes.map(quote =>(
```

Comment:
you'll want to name this something else, as you already have quotes defined in line 9!

Reply:
Maybe you meant it to be quotesElement?